### ISSUE TITLE
unused items doesn't work for lone druid

### ISSUE THREAD

**Issue by Turmfalke2** [yasp-dota/yasp#834]
hi,
Items used on the bear don't count for the unused items message on the analyse page.

**Comment by howardchung** [comment]
Yeah, I actually don't know how to solve this aside from just not computing it at all for lone druid.  The combat log does not report items used from bear.
